src/main/java/org/repodriller/persistence/csv/CSVFile.java

```diff
@@ -47,11 +48,13 @@ public CSVFile(String path, String name, boolean append) {
 
 	@Override
 	public synchronized void write(Object... line) {
-		
+		String field;
 		boolean first = true;
 		for(Object o : line) {
 			if(!first) ps.print(",");
-			ps.print(o);
+			field = o.toString();
+			field = StringEscapeUtils.escapeCsv(field);
```

Comment:
Thank you Mauricio! I was a little dazed and i didn't notice that the method was public.
Hope it's ok now.